code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -24,6 +24,7 @@ var/global/list/image/splatter_cache=list()
 /obj/effect/decal/cleanable/blood/Destroy()
 	for(var/datum/disease2/disease/D in viruses)
 		D.cure(0)
+	viruses = null
```

Comment:
disease/cure() takes and blindly uses a mob reference. If `viruses` actually contains any diseases, it will runtime. Also, this doesn't delete the datums, only dereference the list.